pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -350,7 +351,12 @@ public Representation post(Representation entity) {
 
           // 3/ Request is parsed by the handler which generates a
           // list of FileItems
-          items = upload.parseRequest(getRequest());
+          try {
+            items = upload.parseRequest(getRequest());
+          } catch (FileUploadBase.IOFileUploadException e) {
+            LOGGER.info("Exception uploading segment: " + e);
```

Comment:
Use comma instead of +. That gives full exception stack. That said, this IOException seems to be error condition. We shouldn't turn this to info
I believe there are other exception types thrown here like InvalidContentTypeException that should return 415.